README.md

```diff
@@ -121,7 +121,7 @@ Examples: see [example/angular1](https://github.com/google/dart-scissors/tree/ma
   ```
 
 Valid settings:
-- `pruneCss` (boolean): by default, `true` in `release` only
+- `pruneCss` (`skip` or `print` or `overwrite`): by default, `overwrite` in `release` only
```

Comment:
Also, can you please document each enum value in the readme with the same format as the other enums?